app/tests/spec/models/auth_brokers/fx-ios-v1.js

```diff
@@ -126,23 +130,135 @@ define(function (require, exports, module) {
       });
 
       describe('afterCompleteSignInWithCode', () => {
-        let account;
-
         beforeEach(() => {
           sandbox.spy(broker, 'afterCompleteSignInWithCode');
           sandbox.spy(broker, '_notifyRelierOfLogin');
-          sandbox.spy(FxDesktopV1AuthenticationBroker.prototype, 'afterCompleteSignInWithCode');
-          account = new Account({
-            uid: 'uid'
-          });
+          sandbox.spy(FxiOSAuthenticationBroker.prototype, 'afterCompleteSignInWithCode');
+
           return broker.afterCompleteSignInWithCode(account);
         });
 
         it('broker calls correct methods', () => {
-          assert.isTrue(FxDesktopV1AuthenticationBroker.prototype.afterCompleteSignInWithCode.called);
+          assert.isTrue(broker.afterCompleteSignInWithCode.called);
           assert.isTrue(broker._notifyRelierOfLogin.called);
         });
       });
     });
+
+    describe('createChannel', function () {
+      it('creates a channel', function () {
+        assert.ok(broker.createChannel());
+      });
+    });
+
+    describe('channel errors', function () {
+      it('are propagated outwards', function () {
+        var channel = broker.createChannel();
+
+        var errorSpy = sinon.spy();
+        broker.on('error', errorSpy);
+
+        var error = new Error('malformed message');
+        channel.trigger('error', error);
+        assert.isTrue(errorSpy.calledWith(error));
+      });
+    });
+
+    describe('afterLoaded', function () {
+      it('sends a `loaded` message', function () {
+        sinon.spy(broker, 'send');
+
+        return broker.afterLoaded()
+          .then(function () {
+            assert.isTrue(broker.send.calledWith('loaded'));
+          });
+      });
+    });
+
+    describe('afterSignIn', function () {
+      it('notifies the channel of login, halts by default', function () {
+        sinon.spy(broker, 'send');
+
+        return broker.afterSignIn(account)
+          .then(function (result) {
+            assert.isTrue(broker.send.calledWith('login'));
+            assert.isTrue(result.halt);
+          });
+      });
+    });
+
+    describe('afterSignInConfirmationPoll', () => {
+      it('halts by default', () => {
+        return broker.afterSignInConfirmationPoll(account)
+          .then((result) => {
+            assert.isTrue(result.halt);
```

Comment:
With the changes to the defaultBehavior for `afterSignInConfirmationPoll`, this will start failing. The test will need to be updated accordingly.